cmd/controller/main.go

```diff
@@ -96,6 +96,13 @@ func main() {
 		panic(err.Error())
 	}
 
+	if controller.GetVersion().GitCommit != "" {
```

Comment:
We are checking exact same thing here as well : https://github.com/noironetworks/aci-containers/blob/36fd98d6054b94c154c7facf1e5598707c167ec0/cmd/controller/main.go#L44 
Why do we need to repeat this ?

Reply:
Need it for the version flag too